spec/requests/shopping_list_items_spec.rb

```diff
@@ -38,7 +38,7 @@
       end
       
       context 'when all goes well' do
-        let(:params) { "{\"shopping_list_item\":{\"description\":\"Corundum ingot\",\"quantity\":5,\"notes\":\"To make locks\",\"list_id\":#{shopping_list.id}}}" }
+        let(:params) { { shopping_list_item: { description: 'Corundum ingot', quantity: 5, notes: 'To make locks' } } }
```

Comment:
I decided to change these strings into POROs and just call `to_json` on them in the `subject` block. That's a lot nicer than having those ugly JSON strings with interpolated data and escaped quotes.